pom.xml

```diff
@@ -394,18 +369,18 @@
               </licenseFamily>
             </licenseFamilies>
             <excludes>
+              <exclude>NOTICE</exclude>
               <exclude>**/.*</exclude>
-              <exclude>**/*.txt</exclude>
-              <exclude>**/*.sh</exclude>
-              <exclude>**/*.log</exclude>
+              <exclude>**/*.json</exclude>
+              <exclude>**/*.sqltemplate</exclude>
+              <exclude>**/compose_env</exclude>
+              <exclude>**/*NOTICE*</exclude>
+              <exclude>**/*LICENSE*</exclude>
```

Comment:
why the pom below is excluded? pom file can have a license header.

Reply:
This is a generated pom as part of shading. Its not part of source release and generated when building. Not sure if we can generate HEADER for this auto-generated file